### ISSUE TITLE
The bot is not online :/

### ISSUE THREAD

**Issue by qdweer3** [Skaytacium/ShowBot#1]

**Comment by qdweer3** [comment]
Make the bot online :/

**Comment by Skaytacium** [comment]
Yeah, I'll try to keep it online more often, it's still in development, plus I've mostly achieved what I wanted to with the bot. Might stop maintaining it.
